src/beautifiers/prettydiff.coffee

```diff
@@ -109,6 +110,8 @@ module.exports = class PrettyDiff extends Beautifier
           lang = "scss"
         when "TSS"
           lang = "tss"
+        when "Golang Template"
+          lang = "gohtml"
```

Comment:
@prettydiff please confirm this is correct? Or does it use another `lang` value?